PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -143,6 +160,33 @@ struct PseudorapidityDensityMFT {
         LOGF(fatal, "object not found!");
       }
     }
+    if (doprocessCountingCentrality) {
+      registry.add({"Events/Centrality/Selection", ";status;centrality;events", {HistType::kTH2F, {{4, 0.5, 4.5}, CentAxis}}});
+      auto hstat = registry.get<TH2>(HIST("Events/Centrality/Selection"));
+      auto* x = hstat->GetXaxis();
+      x->SetBinLabel(1, "All");
+      x->SetBinLabel(2, "Selected");
+      x->SetBinLabel(3, "Selected INEL>0");
+      x->SetBinLabel(4, "Rejected");
+
+      registry.add({"Events/Centrality/NtrkZvtx", "; N_{trk}; Z_{vtx} (cm); centrality", {HistType::kTH3F, {MultAxis, ZAxis, CentAxis}}});
+      registry.add({"Events/Centrality/NtrkZvtx_gt0", "; N_{trk}; Z_{vtx} (cm); centrality", {HistType::kTH3F, {MultAxis, ZAxis, CentAxis}}});
```

Comment:
We do not need INEL>0 for Pb-Pb, in my code it is a leftover from copying that I intend to remove.

Reply:
Ok I will remove this